### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by mmasciov** [YonsiG/simple_looping#5]
A suggestion is to add comments wherever what's done is not trivial, for future reference.
This mainly applies to the `Cut` [method](https://github.com/YonsiG/simple_looping/blob/main/MyClass.h#L5563) you are using to implement the selection.
E.g., what is done at [this line](https://github.com/YonsiG/simple_looping/blob/main/MyClass.h#L5613)? I know you now know, given the conversation/explanation on Skype, but it may be worth adding comment lines to remember in a week/month/year from today, and explain to us as "reviewers".

**Comment by YonsiG** [comment]
Yes, thank you Mario! I'll add more comments in the code and push with the next commit!
